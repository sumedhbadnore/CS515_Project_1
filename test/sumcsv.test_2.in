Product,Quantity,Price
Laptop,20,1200
Smartphone,50,500
Headphones,100,50
Monitor,30,300
Keyboard,40,40
Mouse,80,20