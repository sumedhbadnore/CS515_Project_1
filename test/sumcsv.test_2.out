Sum Of Column 'Price': 2110.0